### PR TITLE
Updated doc about fabricating email addresses.

### PR DESCRIPTION
**Pull request by mooreds**
This is from this forum post: https://fusionauth.io/community/forum/topic/304/identity-provider-with-no-email/3

### CODE REVIEW

**Comment by mooreds** [comment]
Going to merge this today.

**Comment by mooreds** [comment]
@robotdan

File: site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc [hunk 1]
```diff
@@ -68,6 +68,23 @@ During development if you want to get a better idea of what your IdP is returnin
 console.info(JSON.stringify(jwt, null, 2));
 ----
 
+=== Workarounds
+
+If the JWT from the OIDC identity provider does not come back with a claim called `email`, you can add your own. If the Userinfo response available to you in the lambda has a unique user, you can build a fake email address from it.
```
Comment (robotdan) [inline site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc:73]: > If the JWT from the OIDC identity provider does not come back with a claim called `email`, you can add your own.

Or whatever value is resolved by the `oauth2.emailClaim` configuration parameter. This is `email` by default.  (looks like this is missing from the API at the moment). 

Fixed doc : https://github.com/FusionAuth/fusionauth-site/commit/a0ab699a5b86ddda7fdeedb5735f80251fe75d6f

**Comment by mooreds** [comment]
Hmmm. Interesting. 

So the ability to use the lambda to set the user's email if the configured `email` claim is not present ombined with the ability to change the `oauth2.emailClaim` setting to change which claim contains the user's email means that the inability of a lambda to overwrite this claim is useless.

You can set the `oauth2.emailClaim` to 'madeupclaimwhichwontexist` and then use the lambda to make an email claim anything you want.

This means that the limits in the `limitations` section of this doc aren't really limits at all.

**Comment by robotdan** [comment]
> You can set the oauth2.emailClaim to 'madeupclaimwhichwontexist` and then use the lambda to make an email claim anything you want.

This is correct. 

> This means that the limits in the `limitations` section of this doc aren't really limits at all.

Sort of, you still can't overwrite it, you can just ensure we don't find one in the OIDC response so you can set it. My only hesitation with documenting it is that it is sort of a hack. If you "fabricate" an email address, we'll run the lambda twice.

**Pull request merged**